Documentation/Stage-1-Introduction/Day-2/Index.rst

```diff
@@ -54,10 +59,9 @@ Practical prerequisites
 
 Before you start this day, please have the following things ready:
 
-* Item 1
-* Item 2
-* Item 3
-
+* Practical prerequisites are verified
+* Theoretical prerequisites are verified
+* Theoretical goals of the day are achieved
```

Comment:
Does this day require anything practical? Like any completed action from the day before?